.github/workflows/ci.yml

```
@@ -173,7 +173,7 @@ jobs:
 
       - name: Run `clash-vexriscv-sim` unittests
         run: |
-          cabal run clash-vexriscv-sim:unittests -- -j2
+          cabal run clash-vexriscv-sim:unittests -- -j1
```

```suggestion
          # XXX: Can't run the unit tests with multiple threads due to GDB servers
          cabal run clash-vexriscv-sim:unittests -- -j1
```

Could you open an issue too?

---

Yep, can do.